cypress/integration/specialFeatures/utilities/cookiePrivacyBanner/index.js

```diff
@@ -0,0 +1,44 @@
+import appConfig from '../../../../../src/server/utilities/serviceConfigs';
+import config from '../../../../support/config/services';
+
+export const getPrivacyBanner = (service, variant) =>
+  cy.contains(
+    appConfig[config[service].name][variant].translations.consentBanner.privacy
```

Comment:
This looks good to me.

A (possibly over-engineered) solution to all the duplication would be to add a higher order function that supplies the consent banner translations to your other functions

Something like this:
```
import pipe from 'ramda/src/pipe';

const withTranslations = func => pipe((service, variant) => ({
  service,
  variant,
  translations: appConfig[config[service].name][variant].translations.consentBanner
}), func);
```
which would simplify these 6 functions to be something like this:
```
const getCookieBanner = withTranslations(({ translations }) =>
  cy.contains(translations.cookie.title)
);
```
or when the lambda still needs the service and variant:
```
const getCookieBannerAccept = withTranslations(({ service, variant, translations }) =>
  getCookieBannerContainer(service, variant)
    .find('button')
    .contains(translations.cookie.accept));
```